service/raft/raft_sys_table_storage.cc

```diff
@@ -5,6 +5,7 @@
 /*
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
+#include "service/raft/messaging.hh"
```

Comment:
These are tiny functions and you add a dep on raft/messaging.hh to two more compile units. If you care so much about proper code reuse, I think you should put them into an own header or find a better place. Besides, the implementation of a function declared in <name>.hh should normally reside in <name>.cc which is not the case here, so it adds to a mess.

Reply:
> you add a dep on raft/messaging.hh to two more compile units.

The raft/messaging.hh header is tiny.

> If you care so much about proper code reuse, I think you should put them into an own header or find a better place.

I think it's a good place - converting between address types is related to messaging.

> Besides, the implementation of a function declared in .hh should normally reside in .cc which is not the case here, so it adds to a mess.

Not sure what you mean here. The implementations of those functions reside in a .cc file - in service/raft/raft_rpc.cc